nautobot_chatops_ipfabric/worker.py

```diff
@@ -705,3 +717,61 @@ def get_wireless_clients(dispatcher, ssid=None, snapshot_id=None):
         ],
     )
     return CommandStatusChoices.STATUS_SUCCEEDED
+
+
+@subcommand_of("ipfabric")
+def find_host(dispatcher, filter_key=None, filter_value=None):
+    """Get host information using the inventory host table."""
+    sub_cmd = "find-host"
+
+    if not filter_key:
+        prompt_find_host_filter_keys(f"{BASE_CMD} {sub_cmd}", "Select filter criteria:", dispatcher)
+        return False
+
+    if not filter_value:
+        dispatcher.prompt_for_text(
+            f"{BASE_CMD} {sub_cmd} {filter_key}",
+            f"Enter a specific {filter_key} to filter by:",
+            f"{filter_key.upper()}",
+        )
+        return False
+
+    is_valid_input = inventory_host_func_mapper.get(filter_key)
+    if not is_valid_input(filter_value):
```

Comment:
personal suggestion for better readability (at first glance, I thought `is_valid_input` as a boolean):
```suggestion
    is_valid_input_func = inventory_host_func_mapper.get(filter_key)
    if not is_valid_input_func(filter_value):
```